<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fff;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>107</y>
     <width>561</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>521</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="le_input_ID">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(0, 81, 255);
border-radius:10px;
</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>ID 입력</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_next">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>75</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(13, 140, 251, 255), stop:1 rgba(58, 224, 191));
	border-radius: 20px;
border:1px solid rgb(0, 81, 255);
}
QPushButton:hover {
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 20px;}
QPushButton:pressed {
border: 3px solid rgb(0, 81, 255);
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 20px;}

</string>
      </property>
      <property name="text">
       <string>다 음</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_join">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>75</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(13, 140, 251, 255), stop:1 rgba(58, 224, 191));
	border-radius: 20px;
border:1px solid rgb(0, 81, 255);
}
QPushButton:hover {
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 20px;}
QPushButton:pressed {
border: 3px solid rgb(0, 81, 255);
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 20px;}

</string>
      </property>
      <property name="text">
       <string>회원가입</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="le_input_PW">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(0, 81, 255);
border-radius:10px;
</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>PW 입력</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_prev">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>75</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(13, 140, 251, 255), stop:1 rgba(58, 224, 191));
	border-radius: 20px;
border:1px solid rgb(0, 81, 255);
}
QPushButton:hover {
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 20px;}
QPushButton:pressed {
border: 3px solid rgb(0, 81, 255);
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 20px;}

</string>
      </property>
      <property name="text">
       <string>이 전</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_show_ID">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(0, 81, 255);
border-radius:10px;
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_join2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>75</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(13, 140, 251, 255), stop:1 rgba(58, 224, 191));
	border-radius: 20px;
border:1px solid rgb(0, 81, 255);
}
QPushButton:hover {
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 20px;}
QPushButton:pressed {
border: 3px solid rgb(0, 81, 255);
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 20px;}

</string>
      </property>
      <property name="text">
       <string>회원가입</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_move_main">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>320</y>
        <width>75</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(13, 140, 251, 255), stop:1 rgba(58, 224, 191));
	border-radius: 20px;
border:1px solid rgb(0, 81, 255);
}
QPushButton:hover {
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 20px;}
QPushButton:pressed {
border: 3px solid rgb(0, 81, 255);
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 20px;}

</string>
      </property>
      <property name="text">
       <string>완 료</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PW</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>311</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color:white;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="lb_login">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>굴림체</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(13, 140, 251, 255), stop:1 rgba(58, 224, 191));
	border-radius: 10px;
border:1px solid rgb(0, 81, 255);</string>
       </property>
       <property name="text">
        <string>회원가입</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_input_id">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>160</width>
         <height>30</height>
        </rect>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(0, 81, 255);
border-radius:10px;
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>ID 입력</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_input_pw">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>240</width>
         <height>30</height>
        </rect>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(0, 81, 255);
border-radius:10px;
</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>비밀번호 입력</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_check_pw">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>190</y>
         <width>240</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(0, 81, 255);
border-radius:10px;
</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>비밀번호 확인</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_input_name">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>240</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(0, 81, 255);
border-radius:10px;
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>이 름</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_check_id">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>90</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(13, 140, 251, 255), stop:1 rgba(58, 224, 191));
	border-radius: 10px;
border:1px solid rgb(0, 81, 255);
}
QPushButton:hover {
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 10px;}
QPushButton:pressed {
border: 3px solid rgb(0, 81, 255);
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 10px;}

</string>
       </property>
       <property name="text">
        <string>중복확인</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_cancle">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>340</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(13, 140, 251, 255), stop:1 rgba(58, 224, 191));
	border-radius: 10px;
border:1px solid rgb(0, 81, 255);
}
QPushButton:hover {
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 10px;}
QPushButton:pressed {
border: 3px solid rgb(0, 81, 255);
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 10px;}

</string>
       </property>
       <property name="text">
        <string>취소</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_join_finish">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>340</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(13, 140, 251, 255), stop:1 rgba(58, 224, 191));
	border-radius: 10px;
border:1px solid rgb(0, 81, 255);
}
QPushButton:hover {
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 10px;}
QPushButton:pressed {
border: 3px solid rgb(0, 81, 255);
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 10px;}

</string>
       </property>
       <property name="text">
        <string>완료</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_id">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>74</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_pw">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>127</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>비밀번호</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_check_pw">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>177</y>
         <width>90</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>비밀번호 확인</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_nick">
       <property name="geometry">
        <rect>
         <x>27</x>
         <y>226</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>이름</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_phonenum">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>294</y>
         <width>240</width>
         <height>30</height>
        </rect>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(0, 81, 255);
border-radius:10px;
</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>핸드폰 번호</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_phonenum">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>280</y>
         <width>80</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>핸드폰 번호</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_login_img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lb_login_img</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
